website/docs/docs/build/python-models.md

```diff
@@ -517,6 +567,30 @@ def model(dbt, session):
 
 </TabItem>
 
+<TabItem value="BigQuery Dataframe">
+
+<File name='models/my_python_model.py'>
```

Comment:
Does bigframes not support `numpy`? If it does, can we align with the other examples using it?